cronutils/error_handler.py

```diff
@@ -113,10 +113,10 @@ def __init__(self, sentry_dsn, descriptor=None, data_limit=100, sentry_client_kw
             sentry_report_limit=0
     ):
         if sentry_client_kwargs:
-            self.sentry_client = SentryClient(dsn=sentry_dsn, **sentry_client_kwargs)
+            sentry_sdk.init(sentry_dsn, **sentry_client_kwargs)
         else:
-            self.sentry_client = SentryClient(dsn=sentry_dsn)
-            
+            sentry_sdk.init(sentry_dsn)
```

Comment:
sentry_sdk may have global config.  See if we can make the DSN optional and skip calling `init` if no params are passed.